Public/Src/Engine/Processes/SharedOpaqueSideband.cs

```diff
@@ -7,25 +7,90 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
+using System.Threading;
 using BuildXL.Pips.Operations;
 using BuildXL.Utilities;
 using BuildXL.Utilities.Collections;
-using BuildXL.Utilities.Tasks;
 using JetBrains.Annotations;
 
+using static BuildXL.Processes.SidebandUtils;
+
 namespace BuildXL.Processes
 {
+    /// <nodoc />
+    internal static class SidebandUtils

```

Comment:
Nit: What about creating a sideband folder and put the classes e.g SidebandUtils, SidebandMetaData in separate files? (not a big fan of all in one file internal / private classes). Feel free to ignore.

Reply:
will do

Reply:
separated